src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -85,6 +99,11 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.AnalyticsEnabled"/>
         public const string GlobalAnalyticsEnabled = "DD_TRACE_ANALYTICS_ENABLED";
 
+        /// <summary>
+        /// Configuration key for a list of tags to be applied globally to spans.
+        /// </summary>
```

Comment:
Wasn't sure on wording for overriding `DD_TRACE_GLOBAL_TAGS`. Any suggestions?

Reply:
`DD_TRACE_GLOBAL_TAGS is deprecated, please use DD_TAGS` ?

Reply:
See [this comment](https://github.com/DataDog/dd-trace-dotnet/pull/716/files#r420154552).

Reply:
Agreed with Lucas's comment.

Something we also want to check here: if folks are still using `DD_TRACE_GLOBAL_TAGS` to set `env`, but they then use `DD_VERSION` and `DD_SERVICE` for version/service, can they do so?

We want to make sure things are backwards compatible / mixable for the config options that don't explicitly override each other. Otherwise it'll be tougher for folks to migrate. I think it's fine though to be explicit about `DD_TAGS` overriding anything of the same name in `DD_TRACE_GLOBAL_TAGS`.

cc @brettlangdon

Reply:
Applied Lucas's suggestion in 1c1b222185b9dbeacf6849d99d8d5181dfe888c9

Reply:
Sorry @jdgumz , I didn't see your reply until I commented and refreshed. They can use a combination of settings like you were saying: the user can set `env` via `DD_TRACE_GLOBAL_TAGS`, and then the remaining settings with `DD_VERSION` and `DD_SERVICE`.

However, the user will only see tags from either `DD_TAGS` (which takes precedence) or `DD_TRACE_GLOBAL_TAGS`, but not both. Is that going to be an issue?

Reply:
I think this behavior is fine.

There are two separate things going on here:

1. Precedence order of `DD_TAGS` to `DD_SERVICE`, `DD_ENV`, and `DD_VERSION`
2. Deprecation and migration of `DD_TRACE_GLOBAL_TAGS` to `DD_TAGS`
  a. Same for `DD_SERVICE_NAME` to `DD_SERVICE`

Reply:
I'm going to resolve this conversation but let me know if it should be re-opened